docs/codeql/codeql-language-guides/customizing-library-models-for-javascript.rst

```diff
@@ -0,0 +1,471 @@
+.. _customizing-library-models-for-javascript:
+
+Customizing Library Models for JavaScript
+=========================================
+
+.. include:: ../reusables/beta-note-customizing-library-models.rst
+
+The JavaScript analysis can be customized by adding library models in data extension files.
+
+A data extension for JavaScript is a YAML file of form:
+
+.. code-block:: yaml
+
+  extensions:
+    - addsTo:
+        pack: codeql/javascript-all
+        extensible: <name of extension point>
```

Comment:
Can you rename _extension point_ to _extensible predicate_? This is the term we are using throughout all our documentation.

Reply:
This has been renamed